<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Regression Suite" duration-ms="19100" started-at="2021-02-21T07:36:18Z" finished-at="2021-02-21T07:36:37Z">
    <groups>
    </groups>
    <test name="login" duration-ms="19100" started-at="2021-02-21T07:36:18Z" finished-at="2021-02-21T07:36:37Z">
      <class name="com.tekarch.SFDC_Login.loginToSF">
        <test-method status="PASS" signature="TriggerDependencies()[pri:0, instance:com.tekarch.SFDC_Login.loginToSF@73ad2d6]" name="TriggerDependencies" is-config="true" duration-ms="4015" started-at="2021-02-20T23:36:14Z" finished-at="2021-02-20T23:36:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TriggerDependencies -->
        <test-method status="PASS" signature="settingUpEnvironment()[pri:0, instance:com.tekarch.SFDC_Login.loginToSF@73ad2d6]" name="settingUpEnvironment" is-config="true" duration-ms="1730" started-at="2021-02-20T23:36:18Z" finished-at="2021-02-20T23:36:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- settingUpEnvironment -->
        <test-method status="FAIL" signature="To_check_Error_For_Invalid_Login()[pri:1, instance:com.tekarch.SFDC_Login.loginToSF@73ad2d6]" name="To_check_Error_For_Invalid_Login" duration-ms="3288" started-at="2021-02-20T23:36:20Z" finished-at="2021-02-20T23:36:23Z">
          <exception class="org.openqa.selenium.NoAlertPresentException">
            <message>
              <![CDATA[no such alert
  (Session info: chrome=88.0.4324.150)
Build info: version: '3.141.5', revision: 'd54ebd709a', time: '2018-11-06T11:42:16'
System info: host: 'LAPTOP-38PPD9KJ', ip: '192.168.162.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.150, chrome: {chromedriverVersion: 88.0.4324.96 (68dba2d8a0b14..., userDataDir: C:\Users\MANU-T~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:55617}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 37a3133daf8b0a21ef5e33f4675e0b8e]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoAlertPresentException: no such alert
  (Session info: chrome=88.0.4324.150)
Build info: version: '3.141.5', revision: 'd54ebd709a', time: '2018-11-06T11:42:16'
System info: host: 'LAPTOP-38PPD9KJ', ip: '192.168.162.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.150, chrome: {chromedriverVersion: 88.0.4324.96 (68dba2d8a0b14..., userDataDir: C:\Users\MANU-T~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:55617}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 37a3133daf8b0a21ef5e33f4675e0b8e
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.alert(RemoteWebDriver.java:932)
	at com.tekarch.poLoginSF.poLoginSF.invalidloginToSalesForce(poLoginSF.java:81)
	at com.tekarch.SFDC_Login.loginToSF.To_check_Error_For_Invalid_Login(loginToSF.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoAlertPresentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- To_check_Error_For_Invalid_Login -->
        <test-method status="PASS" signature="settingReqURL()[pri:0, instance:com.tekarch.SFDC_Login.loginToSF@73ad2d6]" name="settingReqURL" is-config="true" duration-ms="321" started-at="2021-02-20T23:36:27Z" finished-at="2021-02-20T23:36:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- settingReqURL -->
        <test-method status="PASS" signature="to_Check_Whether_Login_Happening_Or_Not()[pri:2, instance:com.tekarch.SFDC_Login.loginToSF@73ad2d6]" name="to_Check_Whether_Login_Happening_Or_Not" duration-ms="5993" started-at="2021-02-20T23:36:28Z" finished-at="2021-02-20T23:36:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- to_Check_Whether_Login_Happening_Or_Not -->
        <test-method status="PASS" signature="settingReqURL()[pri:0, instance:com.tekarch.SFDC_Login.loginToSF@73ad2d6]" name="settingReqURL" is-config="true" duration-ms="3833" started-at="2021-02-20T23:36:34Z" finished-at="2021-02-20T23:36:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- settingReqURL -->
      </class> <!-- com.tekarch.SFDC_Login.loginToSF -->
    </test> <!-- login -->
  </suite> <!-- Regression Suite -->
</testng-results>
